src/main/scala/com/atomist/tree/utils/NodeUtils.scala

```diff
@@ -27,4 +27,13 @@ object NodeUtils {
     )
       .map(m => m.invoke(n, args: _*).asInstanceOf[T])
   }
+
+  def hasNoArgMethod[T](n: GraphNode, methodName: String)
+                              (implicit tag: ClassTag[T]): Boolean = {
+    ReflectionUtils.getAllDeclaredMethods(n.getClass).exists(
+      m => m.getName == methodName && m.getParameterCount == 0 && {
+      //println(s"tag=${tag.runtimeClass},rt=${m.getReturnType}")
```

Comment:
Do we want to keep this println and the surrounding block required by having it?